### PR TITLE
feat: make teams location params optional

### PR DESCRIPTION
**Pull request by rexscaria**

### CODE REVIEW

**Comment by rexscaria** [comment]
Sorry missed this. Please include in next release.

File: teams_locations.go [hunk 1]
```diff
@@ -32,10 +32,10 @@ type TeamsLocation struct {
 	Ip                        string                 `json:"ip,omitempty"`
 	Subdomain                 string                 `json:"doh_subdomain"`
 	AnonymizedLogsEnabled     bool                   `json:"anonymized_logs_enabled"`
-	IPv4Destination           string                 `json:"ipv4_destination"`
-	IPv4DestinationBackup     string                 `json:"ipv4_destination_backup"`
-	DNSDestinationIPsID       string                 `json:"dns_destination_ips_id"`
-	DNSDestinationIPv6BlockID string                 `json:"dns_destination_ipv6_block_id"`
+	IPv4Destination           string                 `json:"ipv4_destination,omitempty"`
```
Comment (jacobbednarz) [inline teams_locations.go]: do you want these to be `*string` to omit it entirely vs being empty?

**Comment by rexscaria** [comment]
I will take *string
Thanks


*Rex Scaria | | ***@***.*** ***@***.***> |
+1-480-628-9203*

**Comment by rexscaria** [comment]
I have updated the code to use *string. this is good to merge after validations.

**Pull request merged**